src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -297,6 +301,9 @@
     text: >
         az deployment create --location WestUS --template-file azuredeploy.json  \\
             --parameters @params.json --parameters https://mysite/params.json --parameters MyValue=This MyArray=@array.json
+  - name: Create a deployment at subscription scope from a template-spec
+    text: >
+        az deployment create --location WestUS --template-spec resourceID
```

Comment:
```suggestion
        az deployment create --location WestUS --template-spec "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Resources/templateSpecs/myTemplateSpec/versions/1.0"
```

It would be better to put a real template spec ID here so people can know that `versions/{version}` is needed.